### PR TITLE
Add warning message for tree/plot alerts

### PR DESCRIPTION
**Pull request by steventlamb**

### CODE REVIEW

**Comment by jwalgran** [comment]
+1 works. Nicely abstracted.

File: opentreemap/treemap/templates/treemap/partials/collectionudf.html [hunk 1]
```diff
@@ -44,6 +44,9 @@
     {% endfor %}
     <tr class="placeholder" style="{{ values|yesno:"display: none;," }}"><td>{% trans "No entries yet!" %}</td>
   </table>
+  <p class="text-muted">
```
Comment (maurizi) [inline opentreemap/treemap/templates/treemap/partials/collectionudf.html]: Does this `<p>` tag need to go inside an `{% if udf.model_config.warning_message %}` block?

Comment (steventlamb) [inline opentreemap/treemap/templates/treemap/partials/collectionudf.html]: can't hurt!

**Comment by steventlamb** [comment]
closing in favor of #2181, comments addressed there

**Pull request closed without merging**